helm-ag.el

```diff
@@ -624,7 +624,7 @@ Default behaviour shows finish and result in mode-line."
   (interactive)
   (goto-char (point-min))
   (let ((read-only-files 0)
-        (files-to-lines (make-hash-table))
+        (files-to-lines (make-hash-table :test #'equal))
```

Comment:
The type of this hash key is list, so `equal` must be used as test function

Reply:
Apologies, I understand now. Strings are objects too.